package/run.sh

```diff
@@ -79,15 +79,20 @@ get_k8s_api_version() {
   set +x # don't print the token
   KUBE_TOKEN=$(</var/run/secrets/kubernetes.io/serviceaccount/token)
   api_version=$(curl -sSk \
-  -H "Authorization: Bearer $KUBE_TOKEN" \
-  "https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_PORT_443_TCP_PORT}/version" | jq -r '.major + "." +.minor')
+      -H "Authorization: Bearer $KUBE_TOKEN" \
```

Comment:
are we testing these script changes for RKE1 profile also?

Reply:
I tested in RKE1. Everything ran as expected.